config.js

```diff
@@ -334,6 +313,33 @@ exports.updateFromArgv = function() {
       ["minperiodtimeunranked", "minperiodtimeblitzunranked, --minperiodtimeliveunranked and/or --minperiodtimecorrunranked"]
       ]
     deprecatedArgs.forEach(ar => testDeprecated(...ar))
+
+    let familyArray = []; 
+    function familyArrayFromGeneralArg(generalArg) {
+        familyArray = []; // first, we reinitialize it to empty at every start
+                  // then, for example, will be ["komi", "komiranked", "komiunranked"];
+        let arr = [ "", "ranked", "unranked"]; // our source to inject on "komi" for example
+        for (let e of arr) {
+            e = generalArg + e; // example : e = "komi" + "ranked";
+            familyArray.push(e);
+        }
+    }
```

Comment:
Last line of function could

`return arr;`

Reply:
oh i see, 

yes indeed we can,
but if we do that we may call the functions more times than needed, so maybe storing the array result in a variable is more ressource efficient ? i really dont know, this is just a question :+1:

Reply:
yes, maybe thats a good idea

i'll try it when i come back later today

Reply:
done by dorus code :+1: